.github/workflows/test-package.yml

```diff
@@ -65,7 +73,7 @@ jobs:
         env:
           PACKAGE_URL: ${{ steps.repo-info.outputs.url }}
         run: |
-          if ! npm install --no-audit; then
+          if ! ${{ inputs.install-command }}; then
```

Comment:
[suggest] How about using `eval` like this?

```diff
         env:
           PACKAGE_URL: ${{ steps.repo-info.outputs.url }}
+          INSTALL_COMMAND: ${{ inputs.install-command }}
```

```suggestion
          if ! eval "$INSTALL_COMMAND"; then
```

Reply:
This seems to work great!
Is there a specific benefit to `eval` in this context?

Reply:
Thanks for addressing the review.

I prefer environment variables over embedded `${{ }}` expressions because:
- they may cause command injections
- local test a shell script is easier